vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -300,6 +302,22 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 				},
 			},
 		},
+		"usb_controller": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Description: "A specification for a USB controller on the virtual machine.",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"usb_version": {
```

Comment:
Can we use `version` here like `vtpm` does?

Reply:
updated to version